ethcore/src/client/client.rs

```diff
@@ -2073,15 +2073,16 @@ impl IoClient for Client {
 				let first = queued.write().1.pop_front();
 				if let Some((header, block_bytes, receipts_bytes)) = first {
 					let hash = header.hash();
-					client.importer.import_old_block(
+					let result = client.importer.import_old_block(
```

Comment:
Just to clarify that understand this correct, essentially `map_or` is used the inverse way here right?

Prints a error message when the Result is `Ok` and don't care when it is `Err`?

Reply:
Yes, exactly. We could change only `.ok()` to `.err()` and it would work, but I thought it was clearer using `if let Err`.

Reply:
Yeah, that is fine!